website/static/js/profile.js

```diff
@@ -731,6 +731,14 @@ SocialViewModel.prototype.unserialize = function(data) {
     var self = this;
     var websiteValue = [];
     $.each(data || {}, function(key, value) {
+        if (key === 'profileWebsites') {
+            value = value.map(function(website) {
+                return $osf.decodeText(website);
+            });
+        } else if (typeof value === 'string') {
```

Comment:
@Nesiehr, I think @alexschiller wanted to point out that this string check is also done in the $osf.decodeText helper, hence not needed here. See [osfHelpers.js#L369](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/static/js/osfHelpers.js#L369)